pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md

```diff
@@ -0,0 +1,61 @@
+
+## Improved window management for Android mobile apps
+
+Starting in Chrome OS 93, improved window compatibility for mobile apps is now available on Android 11-enabled Chromebooks. With this first in a series of upcoming updates, apps will run in a window locked to phone or tablet orientations.
+
+Users can easily [switch between a predefined phone and tablet view](https://support.google.com/chromebook/thread/124854535/introducing-chrome-os-93-to-the-stable-channel?hl=en) by using a simple window resize UI that appears when they hover over the top of the app screen. Users can also disable this window management feature to enable full resizability by clicking the “Resizable” option.
+
+![](/images/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os/window-management.jpg)
+
+This feature applies to most apps available in the Play Store not currently optimized for desktop use. Games and Media apps that have fullscreen mode enabled to [deliver more immersive experiences](https://developer.android.com/training/system-ui/immersive) will continue to launch in fullscreen mode, but users can change the window view at any time.
+
+For more details and tips, check out our post about [optimizing your Android apps for desktops.](/{{locale.code}}/android)
+
+## Check your app’s performance on desktop
+
+Here’s how you can get a better idea of how your users will experience your app on Chrome OS:
+
+- Make sure your Android device is running on version 11 or newer. You can check the Android version on your Chrome OS device under Settings > Apps > Google Play Store > Manage Android Preferences > System > About Device.
+- Confirm that the Chrome OS version of your device is 93 or later. You can check the Chrome OS version of your device under Settings > About.
+- If your app is already installed on your Chrome OS device, uninstall and reinstall it.
+- Launch the application on your Chrome OS device.
+- Ensure that the application is in freeform window mode (not maximized or fullscreen).
+- Check if the top bar of the app has the UI element to switch window orientations.
+
+!!! aside.message--note
+**Note:** This window management feature applies for apps installed via Play Store, not adb (Android Debug Bridge).
+!!!
+
+## Adapt your Android apps for Chrome OS
+
+Better window management for Android mobile apps is an important step toward creating a seamless user experience on larger screens. But the best way to make sure users get the most out of your app on Chrome OS is to adapt it for desktop use.
+
+We broke down some [helpful steps](https://chromeos.dev/en/posts/optimizing-android-app-experiences-for-chromeos) to ensure your app looks and works great across form factors and screen sizes, as well as some [tips for optimizing for Chrome OS.](/{{locale.code}}/android)
```

Comment:
```suggestion
We broke down some [helpful steps](/{{locale.code}}}/posts/optimizing-android-app-experiences-for-chromeos) to ensure your app looks and works great across form factors and screen sizes, as well as some [tips for optimizing for Chrome OS.](/{{locale.code}}/android)
```

Reply:
Good catch! I appreciate the attention to detail!